### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by mohamedelshamy55**
Run a program using the command line.
Use Ruby syntax for basic programming operations.
Use Ruby to solve basic coding challenges.

### CODE REVIEW

File: morse.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+print 'Type some morse code here: '
+morse_str = gets.chomp
```
Comment (Dipeshtwis) [inline morse.rb]: - [x] I would recommend removing these lines and also all the puts line below from 34. As you don't need to ask to enter any, instead we will check directly.

File: README.md [hunk 2]
```diff
@@ -1,57 +1,48 @@
...
+Clone the project by using terminal:
 
-### Setup
+```
+git clone https://github.com/mohamedelshamy55/Morse-code.git
 
-### Install
 
-### Usage
+ruby morse.rb
```
Comment (Dipeshtwis) [inline README.md]: - [x] Here you have missed one step that takes me to the right folder where I can run the project. Include `cd Morse-code` after line 20.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @mohamedelshamy55 & @Mucyosoda, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (mohamedelshamy55) [reply README.md]: thanks for your comment

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @mohamedelshamy55 and @Mucyosoda,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mohamedelshamy55** [comment]
@Dipeshtwis @TedLivist Thank you for finding time to review my project. I appreciate.

**Pull request merged**